### PR TITLE
BEM

### PR DESCRIPTION
**Pull request by alper92**
Hi there,
I tried using BEM. Can someone help me to make my CSS better? Thanks a lot!

### CODE REVIEW

File: components/profile.css [hunk 1]
```diff
@@ -27,9 +27,14 @@
     flex-wrap: wrap;
     align-items: center;
     gap: 3rem;
+    margin: 4rem 2rem;
 }
     
 .profile-statistics i {
     font-size: 2rem;
-    color: orange;
+    color: navy;
```
Comment (milodevhh) [inline components/profile.css:41]: when you what to change the colors you can use root - like that: 
in the globals.css file 
:root {
  --primary-color: #84a63c;
  --secondary-color: #b7d93d;
  --tertiary-color: #667303;
}
You can pick for sure other colors - make sure that you switch the Color and use var in the different classes- 
for example - color: var(--secondary-color);

**Review by milodevhh — COMMENTED** [review]
I hope my feedback helps, we can talk about that

File: index.html [hunk 2]
```diff
@@ -31,11 +31,11 @@ <h2 class="card__question">What property flips the axes in flexbox?</h2>
                 </ul>
             </section>
             <footer class="footer">
```
[the remark this answers is not in the document]
Comment (alper92) [reply index.html:65]: Good Idea...

File: index.html [hunk 3]
```diff
@@ -30,6 +30,38 @@ <h2 class="card__question">What property flips the axes in flexbox?</h2>
                     <li class="card__hashtags__item">css</li>
                 </ul>
             </section>
+            <section class="card">
+                <h2 class="card__question">Question</h2>
+                <span class="card__bookmark card__bookmark--unmarked">
+                    <i class="fa-solid fa-bookmark"></i>
```
[the remark this answers is not in the document]
Comment (alper92) [reply index.html:36]: Hi Milo, I checked your comment. I need the extra class for the modifier. The class "card__bookmark" positions the bookmark and the class "card__bookmark--unmarked" and "card__bookmark-marked" are defining the status.

File: components/profile.css (hunk 1 above)
Comment (alper92) [reply components/profile.css:41]: I guess I need to put that in the global.css. Good idea...

**Pull request closed without merging**